etc/profile-a-l/bcompare.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for Beyond Compare by Scooter Software
+# Description: directory and file compare utility
+# Disables the network, which only impacts checking for updates.
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include bcompare.local
+# Persistent global definitions
+include globals.local
+
+# In case the user decides to include disable-programs.inc, still allow KDE's Gwenview to view images via right click -> Open With -> Associated Application
```

Comment:
```suggestion
# In case the user decides to include disable-programs.inc, still allow
# KDE's Gwenview to view images via right click -> Open With -> Associated Application
```

break long lines